src/main/java/com/wasilewskyy/medical_clinic/service/PatientServiceJPA.java

```diff
@@ -13,8 +16,8 @@ public class PatientServiceJPA {
 
     private final PatientRepositoryJPA patientRepositoryJPA;
 
-    public List<Patient> getAllPatients() {
-        return patientRepositoryJPA.findAll();
+    public Page<Patient> getAllPatients(Pageable pageable) {
```

Comment:
Tutaj zastanow sie czy chcesz zwraca Pagea czy List jak List to na Page<Patient> wywołaj metodę .getContent();